components/camel-wordpress/src/main/docs/wordpress-component.adoc

```diff
@@ -0,0 +1,79 @@
+== Wordpress Component
+
+*Available as of Camel version 2.20.1*
+
+Camel component for https://developer.wordpress.org/rest-api/reference/[Wordpress API].
+
+=== Usage
+
+Currently only the **Posts** and **Users** operations are supported.
+
+==== Configure your route
+
+The `WordpressConfiguration` class can be used to set initial properties configuration to the component instead of passing it as query parameter. The following listing shows how to set the component to be used in your routes.
+
+[source,java]
+--------------------------------------------------
+public void configure() {
+    final WordpressConfiguration configuration = new WordpressConfiguration();
+    final WordpressComponent component = new WordpressComponent();
+    configuration.setApiVersion("2");
+    configuration.setUrl("http://yoursite.com/wp-json/");
+    component.setConfiguration(configuration);
+    getContext().addComponent("wordpress", component);
+    
+    from("wordpress:post?id=1")
+      .to("mock:result");
+}
+--------------------------------------------------
+==== Path Parameters
```

Comment:
Like the other components you need to add the placeholders for component options and endpoint options

// component options: START
// component options: END

// endpoint options: START
//endpoint options: END

The docs will be generated automatically